### PR TITLE
Rename contrib/zenith & zenith_test_utils to neon. Rename custom GUCs.

### PR DESCRIPTION
**Pull request by lubennikovaav**
#1492 

PR to trigger CI

Rename contrib/zenith to contrib/neon. 
Rename custom GUCs:
- zenith.page_server_connstring -> neon.pageserver_connstring
- zenith.zenith_tenant -> neon.tenantid
- zenith.zenith_timeline -> neon.timelineid
- zenith.max_cluster_size -> neon.max_cluster_size

Rename 'wal_acceptors' GUC to 'safekeepers'

### CODE REVIEW

File: compute_tools/tests/pg_helpers_tests.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ mod pg_helpers_tests {
...
+            "fsync = off\nwal_level = replica\nhot_standby = on\nwal_acceptors = '127.0.0.1:6502,127.0.0.1:6503,127.0.0.1:6501'\nwal_log_hints = on\nlog_connections = on\nshared_buffers = 32768\nport = 55432\nmax_connections = 100\nmax_wal_senders = 10\nlisten_addresses = '0.0.0.0'\nwal_sender_timeout = 0\npassword_encryption = md5\nmaintenance_work_mem = 65536\nmax_parallel_workers = 8\nmax_worker_processes = 8\nneon.neon_tenant = 'b0554b632bd4d547a63b86c3630317e8'\nmax_replication_slots = 10\nneon.neon_timeline = '2414a61ffc94e428f14b5758fe308e13'\nshared_preload_libraries = 'neon'\nsynchronous_standby_names = 'walproposer'\nneon.page_server_connstring = 'host=127.0.0.1 port=6400'"
```
[the remark this answers is not in the document]
Comment (lubennikovaav) [reply compute_tools/tests/pg_helpers_tests.rs]: Done.
I also renamed `wal_acceptor_reconnect` to `safekeeper_reconnect`.

I only renamed GUC parameters, but didn't touch table names that are used in console.

**Review by SomeoneToIgnore — COMMENTED** [review]
Looks good.

Besides e2e tests in the cloud repo, looks like the `test_read_validation.py` has to be adjusted? 
> extensions = ["pageinspect", "zenith_test_utils", "pg_buffercache"]

**Comment by SomeoneToIgnore** [comment]
The e2e tests have this in compute logs:

```
thread 'main' panicked at 'pageserver connstr should be provided', compute_tools/src/bin/zenith_ctl.rs:61:10
[507](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:507)
stack backtrace:
[508](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:508)
2022-04-19 13:12:58.232 UTC [http-endpoint] INFO: starting HTTP server on 0.0.0.0:3080
[509](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:509)
2022-04-19 13:12:58.232 UTC [compute-monitor] INFO: watching Postgres activity at postgresql://zenith_admin@localhost/postgres
[510](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:510)

[511](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:511)

[512](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:512)
Compute logs from cluster 'young-dew-142066' and test 'test_creation_stress'
[513](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:513)

[514](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:514)
thread '2022-04-19 13:12:47.679 UTC [compute-monitor] INFO: watching Postgres activity at postgresql://zenith_admin@localhost/postgres
[515](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:515)
main' panicked at 'pageserver connstr should be provided', compute_tools/src/bin/zenith_ctl.rs:61:10
[516](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:516)
stack backtrace:
[517](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:517)
2022-04-19 13:12:47.687 UTC [http-endpoint] INFO: starting HTTP server on 0.0.0.0:3080
[518](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:518)
   0: rust_begin_unwind
[519](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:519)
             at ./rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/std/src/panicking.rs:498:5
[520](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:520)
   1: core::panicking::panic_fmt
[521](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:521)
             at ./rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/panicking.rs:107:14
[522](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:522)
   2: core::panicking::panic_display
[523](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:523)
             at ./rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/panicking.rs:63:5
[524](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:524)
   3: core::option::expect_failed
[525](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:525)
             at ./rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/option.rs:1637:5
[526](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:526)
   4: core::option::Option<T>::expect
[527](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:527)
             at ./rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/option.rs:709:21
[528](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:528)
   5: zenith_ctl::prepare_pgdata
[529](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:529)
             at ./home/circleci/project/compute_tools/src/bin/zenith_ctl.rs:57:30
[530](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:530)
   6: zenith_ctl::main
[531](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:531)
             at ./home/circleci/project/compute_tools/src/bin/zenith_ctl.rs:243:5
[532](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:532)
   7: core::ops::function::FnOnce::call_once
[533](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:533)
             at ./rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/ops/function.rs:227:5
[534](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:534)
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[535](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:535)

[536](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:536)

[537](https://github.com/neondatabase/cloud/runs/6078303737?check_suite_focus=true#step:30:537)
```

**Comment by SomeoneToIgnore** [comment]
So looks like we need to update the cloud code now?
I can help, but still don't get what the issue is exactly, should we ask @\ololobus or somebody else?

**Comment by lubennikovaav** [comment]
> So looks like we need to update the cloud code now? I can help, but still don't get what the issue is exactly, should we ask @\ololobus or somebody else?

Console part is here: https://github.com/neondatabase/cloud/pull/1041
It seem to be OK, but postponed till the next release.

File: compute_tools/src/bin/compute_ctl.rs [hunk 2]
```diff
@@ -116,17 +116,17 @@ fn main() -> Result<()> {
...
         .cluster
         .settings
-        .find("zenith.page_server_connstring")
+        .find("neon.pageserver_connstring")
         .expect("pageserver connstr should be provided");
     let tenant = spec
         .cluster
         .settings
-        .find("zenith.zenith_tenant")
+        .find("neon.tenantid")
```
Comment (SomeoneToIgnore) [inline compute_tools/src/bin/compute_ctl.rs:124]: NIT: A bit odd to have those not in `tenant_id`/`timeline_id` form.

**Review by SomeoneToIgnore — APPROVED** [review]

File: compute_tools/src/bin/compute_ctl.rs (hunk 2 above)
Comment (ololobus) [reply compute_tools/src/bin/compute_ctl.rs:124]: +1

**Review by ololobus — APPROVED** [review]

File: compute_tools/src/compute.rs [hunk 3]
```diff
@@ -256,7 +256,28 @@ impl ComputeNode {
...
+        let mut client = match Client::connect(&self.connstr, NoTls) {
+            Err(e) => {
+                info!(
+                    "cannot connect to postgres: {}, retrying with `zenith_admin` username",
+                    e
+                );
+                let zenith_admin_connstr = self.connstr.replacen("cloud_admin", "zenith_admin", 1);
+
+                let mut client = Client::connect(&zenith_admin_connstr, NoTls)?;
+                client.simple_query("ALTER USER zenith_admin RENAME TO cloud_admin")?;
```
Comment (ololobus) [inline compute_tools/src/compute.rs:273]: @lubennikovaav, ugh, I just realized that we cannot do this rename :(

```
=# alter role alexk rename to ak;
ERROR:  session user cannot be renamed
```

You cannot rename the user used in the active session. We can try to do the following, though:
```
create role cloud_admin with login;
grant zenith_admin to cloud_admin;
```

Comment (ololobus) [reply compute_tools/src/compute.rs:273]: And also I figured out how to test it! Just for test you can use `cloud_superadmin` in the cloud repo and do the additional role create `cloud_admin` -> `cloud_superadmin` in the `compute_ctl`. That way you will test more or less the same process that will happen on prod and staging

Comment (vadim2404) [reply compute_tools/src/compute.rs]: @ololobus, do we still need this?

**Pull request merged**